src/telemetry/telemetryService.ts

```diff
@@ -352,4 +368,8 @@ export default class TelemetryController {
   trackDocumentUpdated(source: string, success: boolean): void {
     this.track(TelemetryEventTypes.DOCUMENT_UPDATED, { source, success });
   }
+
+  trackOpenMongoDBDocumentFromPlayground(source: string): void {
```

Comment:
Maybe we can make `source` here use the `DOCUMENT_SOURCE_PLAYGROUND` and `DOCUMENT_SOURCE_TREEVIEW` types